tests/DependencyInjection/SonataAdminExtensionTest.php

```diff
@@ -17,6 +17,15 @@
 
 class SonataAdminExtensionTest extends AbstractExtensionTestCase
 {
+    public function testContainerCompileWithJMSDiExtraBundle()
+    {
+        $this->container->setParameter('kernel.bundles', [
+            'JMSDiExtraBundle' => true,
+        ]);
+
```

Comment:
Looks like this test causes [a deprecation](https://travis-ci.org/sonata-project/SonataAdminBundle/jobs/317669532#L826)

> Automatic registration of annotations is deprecated since 3.14, to be removed in 4.0: 1x

Please fix it.

Reply:
Looks like you cannot avoid it when testing this piece of code: https://github.com/sonata-project/SonataAdminBundle/blob/ee81056dbd5e2f9f5f962e88340a8a8ed4178838/src/DependencyInjection/SonataAdminExtension.php#L341

Please add the `@group legacy` annotation to your test, it will silence it.

Reply:
@greg0ire Added `@group legacy`

Reply:
Perfect 👍 it should make this job green: https://travis-ci.org/sonata-project/SonataAdminBundle/jobs/317680418